src/models/order.js

```diff
@@ -0,0 +1,29 @@
+import Backbone from 'backbone';
+
+const Order = Backbone.Model.extend({
+  initialize(params) {
+    this.buy = params.buy;
+    this.targetPrice = params.targetPrice;
+    this.symbol = params.symbol;
```

Comment:
Backbone should copy all this data into the attributes automatically, which you can then access with `this.get('targetPrice')`.